VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/hearing-controls/hearing-controls.component.scss

```diff
@@ -67,10 +77,71 @@
 }
 
 #hearing-monitor-div {
-  float: left;
-  padding: 0;
+  float: right;
 }
 
 #judgeControlsDiv {
   padding-right: 0;
 }
+
+.icons-container {
+  width: fit-content;
+  height: 35px;
+  display: grid;
+  grid-template-columns: 60px 60px 60px 60px;
+  grid-template-rows: auto;
+  grid-column-gap: 8px;
+  justify-content: center;
+  align-content: center;
+  background: rgba(0, 0, 0, 0.5);
+  border-radius: 24px;
+  padding: 8px 0px 6px 32px;
+}
+
+.toggle-self-view-icon-btn {
+  grid-column: 1;
+  grid-row: 1;
+  opacity: 50%;
+}
+
+.toggle-self-view-icon-btn:hover {
+  opacity: 100%;
+}
+
+.toggle-audio-mute-icon-btn {
+  grid-column: 2;
+  grid-row: 1;
+  opacity: 50%;
+}
+
+.toggle-audio-mute-icon-btn:hover {
+  opacity: 100%;
+}
+
+.toggle-video-mute-icon-btn,
+.pause-hearing-icon-btn {
+  grid-column: 3;
+  grid-row: 1;
+  opacity: 50%;
+}
+
+.toggle-video-mute-icon-btn:hover,
```

Comment:
could these also be here?
.toggle-hand-raised-icon-btn:hover,
.end-hearing-icon-btn:hover

Reply:
toggle-audio-mute-icon-btn:hover

Reply:
or is something like this possible? - not tested
img[class^="toggle-"]:hover {
opacity: 100%;
}

Reply:
I'll try and see if I can create a base class which extends. I forgot to come back to this.